tests/end-to-end/logging/log-junit-to-file.phpt

```diff
@@ -26,7 +26,7 @@ unlink($logfile);
 --EXPECTF--
 <?xml version="1.0" encoding="UTF-8"?>
 <testsuites>
-  <testsuite name="PHPUnit\SelfTest\Basic\StatusTest" file="%sStatusTest.php" tests="12" assertions="4" errors="2" failures="2" skipped="5" time="%f">
+  <testsuite name="PHPUnit\SelfTest\Basic\StatusTest" file="%sStatusTest.php" tests="12" assertions="4" errors="2" failures="2" skipped="4" time="%f">
```

Comment:
The `skipped` count was wrongfully changed from `4` to `5` in https://github.com/sebastianbergmann/phpunit/commit/9267e537409d7112787d43a2345f810aaf96ffe7 but `4` is indeed the correct value